profiles/bicycle.lua

```diff
@@ -173,21 +173,25 @@ function setup()
 
     surface_speeds = {
       asphalt = default_speed,
+      chipseal = default_speed,
```

Comment:
@mjjbell wdyt? At least adding new surfaces seems to be useful, not sure about changed speeds though.

Reply:
Seems reasonable :+1: